features/ui/page-container/page-container.tsx

```diff
@@ -24,6 +23,7 @@ const Container = styled.div`
 const Main = styled.main`
   position: relative;
   flex: 1;
+  z-index: -2;
```

Comment:
This is the thing why I thought mobile menu icon is not opening because the stack is not defined and resulted not to be seen. Thanks to testing! lol wohoo!